internal/util/logging/console_handler.go

```diff
@@ -54,32 +54,27 @@ type consoleHandler struct {
 	out io.Writer
 }
 
-// groupOrAttrs contains group name or attributes.
-type groupOrAttrs struct {
-	group string
-	attrs []slog.Attr
-}
-
 // newConsoleHandler creates a new console handler.
 func newConsoleHandler(out io.Writer, opts *NewHandlerOpts, testAttrs map[string]any) *consoleHandler {
 	must.NotBeZero(opts)
 
-	return &consoleHandler{
+	h := consoleHandler{
 		opts:      opts,
 		testAttrs: testAttrs,
 		m:         new(sync.Mutex),
 		out:       out,
 	}
+
+	if h.opts.Level == nil {
+		h.opts.Level = slog.LevelInfo
+	}
```

Comment:
Now, we modify the input parameter that is not documented to be modified by this function.

I think we should revert to the previous implementation.